vite.config.ts

```diff
@@ -3,6 +3,6 @@ import react from "@vitejs/plugin-react";
 
 // https://vite.dev/config/
 export default defineConfig({
-	plugins: [react()],
-	base: "/discoirc/",
+  plugins: [react()],
+  base: "./",
```

Comment:
what was the reason for requiring installation in webroot?

Reply:
This is now a relative setup, so it should work anywhere. Someone including us might not want to use `/discoirc/`. I think this is the best way so it can work in any path.